runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java

```diff
@@ -291,21 +310,21 @@ public void onChatMessage(ChatMessage event)
 					double percentage = personalPoints / (totalPoints / 100.0);
 
 					String chatMessage = new ChatMessageBuilder()
-							.append(ChatColorType.NORMAL)
```

Comment:
Probably remove this unnecessary indent

Reply:
I removed the double indents with single on all the appends here.